README.md

```diff
@@ -3,6 +3,8 @@
 [![CircleCI](https://img.shields.io/circleci/project/github/mattermost/mattermost-plugin-agenda/master.svg)](https://circleci.com/gh/mattermost/mattermost-plugin-agenda)
 [![Go Report Card](https://goreportcard.com/badge/github.com/mattermost/mattermost-plugin-agenda)](https://goreportcard.com/report/github.com/mattermost/mattermost-plugin-agenda)
 [![Code Coverage](https://img.shields.io/codecov/c/github/mattermost/mattermost-plugin-agenda/master.svg)](https://codecov.io/gh/mattermost/mattermost-plugin-agenda)
+[![Release](https://img.shields.io/github/v/release/mattermost/mattermost-plugin-agenda)](https://github.com/mattermost/mattermost-plugin-agenda/releases/latest)
```

Comment:
This shows an error because there are only pre-releases. Maybe switch to the pre-release badge?
![Screenshot from 2020-05-27 09-58-45](https://user-images.githubusercontent.com/16541325/82993368-bf691300-a000-11ea-8af2-f5cfc3cc0d9a.png)

Reply:
![image](https://user-images.githubusercontent.com/7575921/83164640-138ff800-a0d2-11ea-8753-efc22400d73d.png)

@levb, @hanzei the was a url query param that was available that includes pre-releases.  They badge will be orange for found pre-releases and turn blue when the first release is done.